### ISSUE TITLE
Virus reported by Bitdefender

### ISSUE THREAD

**Issue by codecypher** [conda-forge/opencv-feedstock#332]

**Comment by jakirkham** [comment]
Moved to the `opencv` feedstock as this appears to be an issue with that package and not Miniforge.

**Comment by jakirkham** [comment]
Just to confirm `libopencv_surface_mathching.4.6.0.dylib` is the actual file name or is that a typo? There is a file called `libopencv_surface_matching.4.6.0.dylib`, but that only has one `h`.

**Comment by mshabunin** [comment]
Reproducible with VirusTotal:
* full 4.6.0 py3.9 package: https://www.virustotal.com/gui/file/dc5d6c46b6a40b5af4a18c879c0e72e22bb02a0c8dea5280c90b8933e2ac1dbc/detection
* surface matching library: https://www.virustotal.com/gui/file/0bdc0e03172658659be64a956ca44ba5a0ac5f9cdb607c0ef40b0539d3032880/detection

Probably a false positive.

**Comment by codecypher** [comment]
Yes, typo. Jeff HolmesOn Oct 19, 2022, at 3:51 AM, jakirkham ***@***.***> wrote:﻿
Just to confirm libopencv_surface_mathching.4.6.0.dylib is the actual file name or is that a typo? There is a file called libopencv_surface_matching.4.6.0.dylib, but that only has one h.
